ecs/filter-parser.conf

```diff
@@ -0,0 +1,21 @@
+[SERVICE]
+    Parsers_File /fluent-bit/parsers/parsers.conf
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name ${PARSER_NAME}
+    Match *
+    Key_Name log
+    Parser json
+    Reserve_Data True
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
```

Comment:
for this config, its just the parser filter, nothing else

Reply:
the idea is that you use this config along with one of the others. The other one will contain the input and output. This one just adds filter.